misc/xml-to-record-converter/src/test/resources/ballerina/sample_26.bal

```diff
@@ -0,0 +1,10 @@
+@xmldata:Name {value: "book"}
+@xmldata:Namespace {prefix: "bk", uri: "http://example.com/book"}
+type Bk_Book record {
+    @xmldata:Namespace {prefix: "bk", uri: "http://example.com/book"}
+    string title;
+    @xmldata:Namespace {prefix: "au", uri: "http://example.com/author"}
+    string title1;
```

Comment:
Is this is suffix acceptable @hasithaa ?

Reply:
Alternative would have to be something like `au_title`, right? I kinda prefer that, but then, it does not conform to our naming convention.

Reply:
This is a unique case. One option to go with the <prepix><Name> combination with camelCase? i.e `auTitle`. Only for conflicting ones. 

btw don't we need `@xmldata:name` for the latter one?

Reply:
We need the @xmldata:name {value: "title"}